digits-server

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 # Copyright (c) 2014-2015, NVIDIA CORPORATION.  All rights reserved.
 
+/usr/bin/pip --upgrade gunicorn
```

Comment:
We definitely don't want to force people to upgrade gunicorn. What is this here for? Do we need to increase the minimum requirement in `requirements.txt`?